### ISSUE TITLE
Is normalization by (x-0.45/0.255) necessary?

### ISSUE THREAD

**Issue by dexter2406** [nianticlabs/monodepth2#339]
I notice in encoder, the `x=(x-0.45)/0.255`, leading the input data range about (-2.0, 2.5), what is the purpose of this? Wouldn't it be more reasonable to make them (-1, 1)? Or just simply leave them as (0, 1) and let the *BatchNorm* do the work?

**Comment by macaodha** [comment]
Please see https://github.com/nianticlabs/monodepth2/issues/12

**Comment by dexter2406** [comment]
> Please see #12

Thanks for the reply.

Okay it's for Imagenet pretrained weights. But if I only use pretrianed weights as a starting point and train my own model, and don't do this normalization from now on, the model will then adapt to the new input range right?

**Comment by mdfirman** [comment]
Of course – in theory your model could adapt to the new range. But why make life hard for the model?

I suppose this initial normalisation is a sort of form of batch norm, applied to the input images. Indeed you could perhaps try replacing it with a proper batch norm layer applied before the first convolution. Or as you say, remove it and let the network figure it. 

If you are training from scratch (i.e. with no pretraining at all) then I could imagine that this normalisation would have little impact and you could indeed safely remove it.
